src/struct_mv/coarsen.c

```diff
@@ -358,7 +308,10 @@ hypre_CoarsenBoxArrayArrayNeg( hypre_BoxArrayArray   *boxaa,
       hypre_ForBoxI(j, boxa)
       {
          hypre_CopyBox(hypre_BoxArrayBox(boxa, j), box);
-         hypre_CoarsenBoxNeg(box, refbox, origin, stride);
+         //hypre_CoarsenBoxNeg(box, refbox, origin, stride);
```

Comment:
Remove the commented line here? Particularly since `hypre_CoarsenBoxNeg()` was removed. Same thing below.